src/pages/Components/Checkbox/index.md

```diff
@@ -9,37 +9,7 @@ order: 1
 
 <preview path="src/pages/Components/Checkbox/previews/intro" nude="true"></preview>
```

Comment:
I think we need a preview structure like the radio preview, users can think that the state indications are the labels of the buttons.

Reply:
I really think that this will change when we will develop the labels, fields and validation, I think it is better to make the change when we will decide a definitive course of action.